DscResources/MSFT_xMySqlServer/MSFT_xMySqlServer.schema.mof

```diff
@@ -2,19 +2,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlServer")] 
 class MSFT_xMySqlServer : OMI_BaseResource
 {
-  [Key, Description("Provides the service name to use during setup of MySQL")]
-  String ServiceName;
-  
-  [Write,
-  ValueMap{"Present", "Absent"},Values{"Present", "Absent"},
-  Description("Ensure resource is present or absent")]
-  String Ensure;
-  
-  [required,
-  Write,
-  EmbeddedInstance("MSFT_Credential"),
-  Description("The root credential that is used to install mySql server.")]
-  String RootPassword;
+    [Key, Description("mySql Version Number")] String MySqlVersion;
```

Comment:
You can still run multiple versions.  It just uses the default service name for them.  Version 5.6.x use MySQL56 and version 5.7.x use MySQL57.